lib/src/json_document_transformer.dart

```diff
@@ -22,7 +22,7 @@ final jsonDocument = new JsonDocumentTransformer();
 /// [FormatException]. If an unencodable object is added to the sink, it
 /// synchronously throws a [JsonUnsupportedObjectError].
 class JsonDocumentTransformer
-    implements StreamChannelTransformer<Object, String> {
+    implements StreamChannelTransformer<dynamic, String> {
```

Comment:
Without it I get a strong mode errors like the following in 3 different places

[error] The argument type 'JsonDocumentTransformer' cannot be assigned to the parameter type 'StreamChannelTransformer<String, dynamic>'. (package:json_rpc_2/src/client.dart, line 53, col 24)
[error] The argument type 'JsonDocumentTransformer' cannot be assigned to the parameter type 'StreamChannelTransformer<String, dynamic>'. (package:json_rpc_2/src/peer.dart, line 48, col 24)
[error] The argument type 'JsonDocumentTransformer' cannot be assigned to the parameter type 'StreamChannelTransformer<String, dynamic>'. (package:json_rpc_2/src/server.dart, line 60, col 24)